### DISCUSSION TITLE
How to Implement Openthread Border Router in RTOS

### DISCUSSION THREAD

**Discussion by tuopu** [openthread/openthread#9940] [General]
Hello expert:
At present, the official offer is to use Raspberry Pi and Linux to implement a boundary router. May I ask if there is a specific implementation plan for some simple applications, such as using embedded operating systems such as Freertos or Zephyr with LWIP to implement a boundary router.

**Comment by jwhui** [comment]
Espressif has an RTOS-based Thread Border Router solution: https://openthread.io/codelabs/esp-openthread-hardware
